src/Symfony/Component/DependencyInjection/ServiceLocator.php

```diff
@@ -26,9 +26,10 @@
  *
  * @template-covariant T of mixed
  *
- * @implements ServiceProviderInterface<T>
+ * @implements ServiceCollectionInterface<T>
+ * @implements \IteratorAggregate<string, T>
```

Comment:
Can be removed as we inherit this information from the `ServiceCollectionInterface`, right?